pkg/controller/registry/reconciler/grpc.go

```diff
@@ -16,6 +17,8 @@ import (
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/ownerutil"
 )
 
+const CatalogSourceUpdateKey = "olm.catalogSource.update"
```

Comment:
I think we should start sticking to the [conventions laid out in the k8s docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) for label keys. WDYT about something like `catalogsource.operators.coreos.com/update`?

Reply:
That sounds good to me